src/components/Footer/Footer.js

```diff
@@ -28,8 +36,19 @@ const Footer = ({
   blockStatus,
   loadingVote,
   showScoreButton = false,
-  onPressScore
+  onPressScore,
+  isShowDM = false
 }) => {
+  const {sendMessageDM} = useDMMessage();
+  const [profile] = React.useContext(Context).profile;
+
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+  const refSheet = React.useRef();
+  const {createSignChat} = useCreateChat();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props and state variables have been introduced to manage the direct messaging feature. The `useDMMessage` hook is used to send direct messages. A loading state is maintained for both regular and anonymous direct messages. No issues found here.
```diff
+  isShowDM = false
+ }) => {
+   const {sendMessageDM} = useDMMessage();
+   const [profile] = React.useContext(Context).profile;
+ 
+   const [loading, setLoading] = React.useState({
+     loadingDm: false,
+     loadingDmAnon: false
+   });
+   const refSheet = React.useRef();
+   const {createSignChat} = useCreateChat();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->